doc/testing-resolvers.md

```
@@ -0,0 +1,438 @@
+- execute other pipeline functions
+- validate the GraphQL schema
+
+What you get back is the **request mapping** (what would be sent to the data source) or the **response mapping** (what would be returned to the client), depending on which function you evaluate.
+
+---
+
+## Unit resolver
+
+A unit resolver has a single `request` function and a single `response` function.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)